oshmem/shmem/c/shmem_set_lock.c

```diff
@@ -27,5 +28,10 @@
 
 void shmem_set_lock(volatile long *lock)
 {
-    _shmem_set_lock((void *)lock, sizeof(long));
+	if (oshmem_shmem_mcs_locks) {
+		SHMEM_API_VERBOSE(10, "shmem_set_lock using MCS Lock");
```

Comment:
maybe such comments are worth to be common for all type of locks? (i.e. to move them to the very beginning of func)

Reply:
Agreed and moved in the latest commit.